src/daml/Daml/Finance/Setup/Scenario/BondIssuance.daml

```diff
@@ -94,7 +94,7 @@ run = do
 
   -- Instruments
   usd   <- originateToken               centralBank centralBank pub epoch "USD"   "United States Dollar"
-  bond1 <- originateZeroCouponBond      registry    issuer                "BOND1" "Zero Coupon Bond 5Y"                         pub (tt $ date 2022 tm td) (date 2022 tm td) (date 2027 Sep td) usd
+  bond1 <- originateZeroCouponBond      registry    issuer                "BOND1" "Zero Coupon Bond 5Y"                         pub (tt $ date 2022 tm td) (date 2022 tm td) (date 2027 tm td) usd
   bond2 <- originateFixedRateBond       registry    issuer                "BOND2" "Fixed Rate Bond 5Y @ 1.1%"                   pub (tt $ date 2020 tm td) (date 2020 tm td) ["EMPTY"] registry (date 2021 tm td) (date 2025 tm td) Act365Fixed Following 0.011 M 12 usd
```

Comment:
This will come back biting at us in 2 years 😅 raising an issue to keep track of it